src/KM_Controls.pas

```diff
@@ -905,6 +906,7 @@   TKMColumnBox = class(TKMControl)
     procedure SetColumns(aHeaderFont: TKMFont; aCaptions: array of string; aOffsets: array of Word);
     procedure AddItem(aItem: TKMListRow);
     procedure Clear;
+    procedure RevertLastItemIndexChange;

```

Comment:
I don't like this function in the controls library, I don't think it will get used anywhere else, it's just to solve this specific issue. Can't we find a better way like not changing it to AI in the first place if the slot is not available?